src/Analyzer.BicepProcessor/BicepTemplateProcessor.cs

```diff
@@ -91,47 +98,42 @@ public static (string, BicepMetadata) ConvertBicepToJson(string bicepPath)
                 throw new Exception($"Bicep issues found:{SysEnvironment.NewLine}{string.Join(SysEnvironment.NewLine, bicepIssues)}");
             }
 
-            string GetPathRelativeToEntryPoint(string absolutePath) => Path.GetRelativePath(
```

Comment:
I think we were getting the relativePath before?

Reply:
It still is - I moved this inline to line 116. Line 101 is simply saving the directory name for the entry point file, which doesn't change.